modules/autofill/autofill-impl/build.gradle

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2021 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'com.squareup.anvil'
+    id 'kotlin-android-extensions'
+}
+
+apply from: "$rootProject.projectDir/gradle/android-library.gradle"
+
+dependencies {
+    implementation project(path: ':app-build-config-api')
+    implementation project(path: ':di')
+    implementation project(path: ':common')
+    implementation project(path: ':common-ui')
+    implementation project(path: ':autofill-api')
+    implementation project(path: ':secure-storage-api')
+    implementation project(path: ':device-auth-api')
+    implementation project(path: ':browser-api')
+    api project(path: ':autofill-store')
```

Comment:
This doesn't need to be an api project. See the comment above for the app build.gradle